SSA/Projects/FullyHomomorphicEncryption/Basic.lean

```diff
@@ -42,7 +42,7 @@ theorem WithBot.npow_coe_eq_npow (n : Nat) (x : ℕ) : (WithBot.some x : WithBot
   done
 
 noncomputable def f : (ZMod q)[X] := X^(2^n) + 1
-
+/-! Charaterizing `f`: `f` is monic of degree `2^n` -/
```

Comment:
```suggestion
/-- Charaterizing `f`: `f` is monic of degree `2^n` -/
```
Comments with the double dash are docstrings that appear when you hover over the theorem.